modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/search/spi/model/index/contributor/SegmentsEntryModelDocumentContributor.java

```diff
@@ -72,15 +71,10 @@ public void contribute(Document document, SegmentsEntry segmentsEntry) {
 	}
 
 	private long[] _getRoleIds(long segmentsEntryId) {
-		List<SegmentsEntryRole> segmentsEntryRoles =
+		return TransformUtil.transformToLongArray(
```

Comment:
@jeffwu0724 You can inline this method, since after your change it contains a single method call. Same for `_getSegmentsEntryIds` in `UserModelDocumentContributor`.